react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+
+  return (
+    <div>
+      <div>{items[current]}</div>
+      <div>
+        <Button
+          text="Prev"
+          onClick={() => setCurrent(((current - 1) + items.length) % items.length)}
+          current={current}
+        />
+        <Buttons
+          count={items.length}
+          onClick={(i) => setCurrent(i)}
+          current={current}
+        />
+        <Button
+          text="Next"
+          onClick={() => setCurrent((current + 1) % items.length)}
+          current={current}
+        />
+      </div>
+    </div>
+  );
+}
+
+/**
+ * A button that toggles its color between white and lightblue.
+ */
+function Button({ text, onClick, current, backgroundColor="white" }) {
```

Comment:
Use single quotes around `white` since this is JS code, not JSX. Only use double quotes when passing component prop values that are strings (not inside `{}`, since it's JS code in there).